From 4f0a335e44fa7ac2bf0f4c6b6d5408f7f50138c8 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Wed, 22 Mar 2017 09:52:13 -0700
Subject: [PATCH] MANATEE-324 pull MANATEE-288 fix into manta-manatee,
 sdc-manatee Reviewed by: David Pacheco <dap@joyent.com> Approved by: David
 Pacheco <dap@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index d6c8218..fedc631 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+ssh://git@github.com:joyent/manatee.git#acbcf534d671d9f75a7c3092520f650bbfb6b803",
+        "manatee": "git+ssh://git@github.com:joyent/manatee.git#86ae52d4a9988958aec3ec9955f3e2190bfa81c7",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
-- 
2.21.0

